setup/jupyter_env

```diff
@@ -4,37 +4,38 @@
-  export MANPATH="/apps/python-3.7.5/share/man:$MANPATH"
-  export LD_LIBRARY_PATH="/apps/python-3.7.5/lib:$LD_LIBRARY_PATH"
+# module load python/3.7.5
+export PATH="/apps/python-3.7.5/bin:$PATH"
+export MANPATH="/apps/python-3.7.5/share/man:$MANPATH"
+export LD_LIBRARY_PATH="/apps/python-3.7.5/lib:$LD_LIBRARY_PATH"
   
-  # module load boost/1.70
-  export LD_LIBRARY_PATH="/apps/boost-1.70/lib:$LD_LIBRARY_PATH"
-  export CPATH="/apps/boost-1.70/include:$CPATH"
-  export LIBRARY_PATH="/apps/boost-1.70/lib:$LIBRARY_PATH"
-  export Boost_INCLUDE_DIR="/apps/boost-1.70/include"
-  export BOOST_ROOT="/apps/boost-1.70"
-  export BOOST_INCLUDE_DIRS="/apps/boost-1.70/include"
-  export BOOST_LIBRARY_DIRS="/apps/boost-1.70/lib"
+# module load boost/1.70
+export LD_LIBRARY_PATH="/apps/boost-1.70/lib:$LD_LIBRARY_PATH"
+export CPATH="/apps/boost-1.70/include:$CPATH"
+export LIBRARY_PATH="/apps/boost-1.70/lib:$LIBRARY_PATH"
+export Boost_INCLUDE_DIR="/apps/boost-1.70/include"
+export BOOST_ROOT="/apps/boost-1.70"
+export BOOST_INCLUDE_DIRS="/apps/boost-1.70/include"
+export BOOST_LIBRARY_DIRS="/apps/boost-1.70/lib"
   
-  # module load gcc/8.3.0
-  export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//gmp-6.1.2/lib:$LD_LIBRARY_PATH"
-  export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//mpc-1.1.0/lib:$LD_LIBRARY_PATH"
-  export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//mpfr-4.0.2/lib:$LD_LIBRARY_PATH"
-  export PATH="/apps/gcc-8/gcc-8.3.0//bin:$PATH"
-  export MANPATH="/apps/gcc-8/gcc-8.3.0//share/man:$MANPATH"
-  export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//lib:$LD_LIBRARY_PATH"
-  export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//lib64:$LD_LIBRARY_PATH"
-  export CC="/apps/gcc-8/gcc-8.3.0//bin/gcc"
-  export CXX="/apps/gcc-8/gcc-8.3.0//bin/g++"
-  export FC="/apps/gcc-8/gcc-8.3.0//bin/gfortran"
+# module load gcc/8.3.0
+export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//gmp-6.1.2/lib:$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//mpc-1.1.0/lib:$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//mpfr-4.0.2/lib:$LD_LIBRARY_PATH"
+export PATH="/apps/gcc-8/gcc-8.3.0//bin:$PATH"
+export MANPATH="/apps/gcc-8/gcc-8.3.0//share/man:$MANPATH"
+export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//lib:$LD_LIBRARY_PATH"
+export LD_LIBRARY_PATH="/apps/gcc-8/gcc-8.3.0//lib64:$LD_LIBRARY_PATH"
+export CC="/apps/gcc-8/gcc-8.3.0//bin/gcc"
+export CXX="/apps/gcc-8/gcc-8.3.0//bin/g++"
+export FC="/apps/gcc-8/gcc-8.3.0//bin/gfortran"
   
-  # module load openmpi/4.0.4
-  export PATH="/apps/openmpi-4.0.4/openmpi-4.0.4/bin:$PATH"
-  export MANPATH="/apps/openmpi-4.0.4/openmpi-4.0.4/share/man:$MANPATH"
-  export LD_LIBRARY_PATH="/apps/openmpi-4.0.4/openmpi-4.0.4/lib:$LD_LIBRARY_PATH"
+# module load openmpi/4.0.4
+export PATH="/apps/openmpi-4.0.4/openmpi-4.0.4/bin:$PATH"
+export MANPATH="/apps/openmpi-4.0.4/openmpi-4.0.4/share/man:$MANPATH"
+export LD_LIBRARY_PATH="/apps/openmpi-4.0.4/openmpi-4.0.4/lib:$LD_LIBRARY_PATH"
+
+# serialbox
```

Comment:
Can we retain the if-condition and put only the serialbox root outside of the if. We should rely on modules outside of the Jupyter environment (when ssh'ing onto a login node) and thus should use `source setup/module_env` there instead of this file.

Reply:
The issue wasn't necessarily just with the serialbox path, the if condition seemed to be preventing notebooks from picking up the environment properly... but I can add it back and we can address in a different PR separate from this if you'd like